about.css

```diff
@@ -409,11 +437,12 @@ a {
 
   .bc-cont {
     margin: 2rem auto;
-    padding: 10rem 0.5rem;
+    padding: 1rem 0.5rem;
+    width: 60%;
   }
 
   .past {
-    padding: 6rem 0.5rem;
+    padding: 2rem 0.5rem;
   }
 
   .foot-text {
```

Comment:
There is a margin at the bottom of your `About` page footer that makes for bad UI/UX.

<img width="1440" alt="Screen Shot 2022-07-27 at 9 11 21" src="https://user-images.githubusercontent.com/82426682/181268942-831d73f8-7958-4745-a2f4-93ae8d987438.png">

Please fix it.